README.md

```diff
@@ -15,6 +15,8 @@
   <h1>RAW Loader</h1>
 </div>
 
+A loader for webpack that lets you import files as a string.

```

Comment:
That could be a subtitle , if they are a prerequisite for Loader README's, like in the webpack README, I will update it accordingly then :).

```
<h1>RAW Loader</h1>
<h3>A loader for webpack that lets you import files as a string.</h3>
```